src/libsync/abstractnetworkjob.cpp

```diff
@@ -64,7 +67,7 @@ AbstractNetworkJob::AbstractNetworkJob(AccountPtr account, const QString &path,
     OC_ASSERT(account != parent);
 
     _timer.setSingleShot(true);
-    _timer.setInterval((httpTimeout ? httpTimeout : 300) * 1000); // default to 5 minutes.
+    _timer.setInterval(httpTimeout.count() ? httpTimeout : 5min); // default to 5 minutes.
```

Comment:
This is a bit confusing to read because you use the `.count()` function in the condition and then use the actual chrono API of QTimer.. no idea how to make it more expressive/semantic tho. 
The code is still correct of course..